### PR TITLE
[STTNHUB-65] - Filter embargoed items from cards

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by petrjasek** [comment]
so what's the idea? display embargoed items by default?

**Comment by akintolga** [comment]
I think idea is to use company type filter for home page cards as well. At the moment it is only used in wire and it is working.

File: newsroom/wire/search.py [hunk 1]
```diff
@@ -282,22 +286,17 @@ def has_permissions(self, item, ignore_latest=False):
...
         if not product:
             return
 
-        query['bool']['must'].append({
-            "bool": {
-                "should": [
-                    {"range": {"embargoed": {"lt": "now"}}},
-                    {"bool": {"must_not": {"exists": {"field": "embargoed"}}}}
-                ]
-            }
```
Comment (petrjasek) [inline newsroom/wire/search.py:299]: so how is this filter applied now?

Comment (akintolga) [reply newsroom/wire/search.py:299]: There's a filter in company_types https://github.com/superdesk/newsroom/blob/stt/settings.py#L61
So this one will be used now. But let me know if you think we need another way.

Comment (petrjasek) [reply newsroom/wire/search.py:299]: I see, np then

**Pull request merged**